src/build/stable/commands/install_global_dependencies/install_dfx.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
```

Comment:
I have these echos in the install scripts. I think they might be helpful. Are they? Should they only come for VERBOSE?

Reply:
They are fine